src/System.Data.SqlClient/src/System/Data/SqlClient/SqlInternalConnectionTds.cs

```diff
@@ -321,6 +355,56 @@ internal SqlConnectionTimeoutErrorInternal TimeoutErrorInternal
             }
         }
 
+        // The erros in the transient error set are contained in
+        // https://azure.microsoft.com/en-us/documentation/articles/sql-database-develop-error-messages/#transient-faults-connection-loss-and-other-temporary-errors
+        private static void populateTransientErrors()
```

Comment:
This should be Pascal Case, e.g. `PopulateTransientErrors`, though, if the set is initialized as commented on above, this method could go away.